### ISSUE TITLE
Sending bitcoin with ledger steps seem out of order

### ISSUE THREAD

**Issue by edgarkhanzadian** [blockstack/blockstack-app#4887]
In ledger flow, it first signs the transaction on ledger and then user clicks on “confirm” button. In stacks flow, user clicks on “confirm” button and then goes through transaction signing on ledger.
Stacks flow seems correct to me and we might want to swap steps on bitcoin.

it seems like when we click on a fee item, we sign a transaction and then present a review of the transaction. When we run code for signing a transaction, on software wallets it does it in a moment and immediately routes user to the confirm screen, ledger though requires a separate flow for that transaction signing. So we just need to move signing code from a fee button to a confirm button

BTC:

https://github.com/leather-wallet/extension/assets/22010816/fffbb149-26de-47ff-a2b3-4cf7f978d225

STX:

https://github.com/leather-wallet/extension/assets/22010816/8c541ab3-ec26-4cf8-9734-a82ecf9fe265

**Comment by markmhendrickson** [comment]
@fabric-8 @kyranjamie I presume it's best that we always confirm for Ledger-based approval at the very end of the signing flow i.e. on the approval screen itself and not earlier / before we get to that screen?

**Comment by kyranjamie** [comment]
It works this way primarily because the signing originally took place when choosing the fee. 

@edgarkhanzadian's correct and I think both in both software and Ledger modes, the signing only takes place once the user confirms (rather than signing it, and only broadcasting it if the user confirms).

**Comment by markmhendrickson** [comment]
@fabric-8 is more design guidance needed here per the Ledger-related designs for the new approval UX, or can we move into dev?

**Comment by fabric-8** [comment]
@markmhendrickson Yes let's move into dev — I'll have to update a few of the lottie assets, but will follow up with that later this week as it's not critical for implementation to get started
